pkg/handler/handler.go

```diff
@@ -234,12 +239,12 @@ func (h handler) unbind(w http.ResponseWriter, r *http.Request, params map[strin
 		return
 	}
 
-	err := h.broker.Unbind(instanceUUID, bindingUUID)
+	resp, err := h.broker.Unbind(instanceUUID, bindingUUID)
 
 	if errors.IsNotFound(err) {
-		writeResponse(w, http.StatusGone, struct{}{})
+		writeResponse(w, http.StatusGone, resp)
 	} else {
-		writeDefaultResponse(w, http.StatusOK, struct{}{}, err)
```

Comment:
Set it to still return the response, but the response is now an empty struct. Will make it easy to change if the return response type ever changes and keeps it consistent with other actions.